packages/react-server-test-pages/pages/styles/promises.js

```diff
@@ -0,0 +1,31 @@
+
+import React from "react";
+
+export default class StylePromises {
+	getHeadStylesheets () {
+		return [
+			delay(1000)
+				.then(() => "./styles/first.css"),
+			delay(100)
+				.then(() => "./styles/second.css"),
```

Comment:
I was lazy and just didn't add any. But per our conversation on slack, I added some: https://github.com/redfin/react-server/pull/115